Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'rest-client' # http
 gem 'mandate', '0.4.0.beta.1'
 
 # Setup dependencies
-gem 'exercism-config', '>= 0.43.0'
+gem 'exercism-config', '>= 0.44.0'
```

Comment:
This changes the config so the field is now `aws_submissions_bucket` not `aws_iterations_bucket`